core/domain/collection_domain_test.py

```diff
@@ -978,187 +1010,192 @@ def setUp(self):
             ['editor_id'], ['viewer_id'], ['contributor_id'], {}, 1, 1,
             current_time, current_time)
 
-    def test_collection_summary_gets_created(self):
+    def test_collection_summary_gets_created(self) -> None:
         self.assertEqual(
             self.collection_summary.to_dict(), self.collection_summary_dict)
 
-    def test_validation_passes_with_valid_properties(self):
+    def test_validation_passes_with_valid_properties(self) -> None:
         self.collection_summary.validate()
 
-    def test_validation_fails_with_invalid_title(self):
-        self.collection_summary.title = 0
-        with self.assertRaisesRegex(
+    def test_validation_fails_with_invalid_title(self) -> None:
+        self.collection_summary.title = 0 # type: ignore[assignment]
```

Comment:
Some of these tests check for right value type assigment. So, if we remove these tests we have backend coverage error  to fix that coverage error we have to remove validate() method, but this method is also used in other files. 
can you please help !!

Reply:
I think you can remove the simple `isinstance` checks from the `validate` funciton.

Reply:
Done!